pkg/expression/builtin.go

```diff
@@ -135,7 +135,7 @@ func newBaseBuiltinFunc(ctx BuildContext, funcName string, args []Expression, tp
 	return bf, nil
 }
 
-func newReturnFieldTypeForBaseBuiltinFunc(funcName string, retType types.EvalType, ec *ExprCollation) *types.FieldType {
+func newReturnFieldTypeForBaseBuiltinFunc(funcName string, retType types.EvalType, ec *ExprCollation) (*types.FieldType, error) {
```

Comment:
I think this change can be reverted. Previously `newReturnFieldTypeForBaseBuiltinFunc` has an error return type, but now it doesn't any more. So may be we should not change it back, unless it is a must.

Reply:
right, it looks like unnecessary. I removed the `error` return and have reverted it.